composer/utils/fx_utils.py

```diff
@@ -111,6 +113,30 @@ def replace_op(gm: GraphModule, src_ops: Union[Callable, str, List[Union[Callabl
     return gm
 
 
+class block_stochastic_module(nn.Module):
```

Comment:
nit: Use CamelCase for class names

```suggestion
class BlockStochasticModule(nn.Module):
```

And docstrings...